frontend/src/features/admin-form/create/payment/PaymentDrawer.tsx

```diff
@@ -148,24 +149,30 @@ export const PaymentInput = (): JSX.Element => {
           )
         },
         validateMin: (val) => {
+          if (minPaymentAmountCents === undefined) return true
           return (
-            Number(val?.trim()) >= minPaymentAmount ||
+            // val is in dollars
+            Number(val?.trim()) >=
+              Number(centsToDollars(minPaymentAmountCents)) ||
             `Please enter a payment amount above ${formatCurrency(
-              minPaymentAmount,
+              Number(centsToDollars(minPaymentAmountCents)),
```

Comment:
Two comments on approach here:

1. the config value `minPaymentAmountCents` is the reference value. When we want to compare whether an input is valid against it, the input should be transformed/normalized to make a legitimate comparison, but the reference value itself should be left alone.

2. Not critical in this case, but still an improvement to the previous code would be to be consistent with how we deal with amounts. Although we are not doing arithmetic operations here, let's be consistent in the thinking that integer arithmetic is safe, while floating point arithmetic is not. And with that in mind, the safe ways to handle the operations here is to convert the input from `dollarsToCents` and then compare to the reference value in cents, and not the other way around.

Same comment for line 167 below.

So Basically:
* No need to do any transformation to `minPaymentAmountCents` and `maxPaymentAmountCents`
* Transform the string input to a cents value before comparing to the min and max thresholds

Reply:
thanks for this! made changes in [663724e](https://github.com/opengovsg/FormSG/pull/5972/commits/663724e45e6f2f4d8a5649106555709ccf657d5a)